scripts/arm32_ci_script.sh

```diff
@@ -178,6 +178,55 @@ function cross_build_corefx {
     ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build-managed.sh -$__buildConfig -skipTests -BuildPackages=false
 }
 
+# Cross builds corefx using Docker image
+function cross_build_corefx_with_docker {
+    # Cross building using Docker image
+    currentWorkingDirectory=`pwd`
+
+    # Build rootfs in docker
+    #dockerCmd="sudo docker run -i --rm -v $currentWorkingDirectory:/opt/corefx -w /opt/corefx chcosta/dotnetcore:ubuntu1404_cross_prereqs_v1"
+    #dockerCmd="sudo docker run -i --rm -v $currentWorkingDirectory:/opt/corefx -w /opt/corefx t2wish/dotnetcore:ubuntu1404_cross_prereqs_v2"
+    # Build rootfs in Docker
+    #if [ "$__buildArch" == "arm" ]; then
+    #    buildRootfsCmd="/opt/corefx/cross/build-rootfs.sh arm"
+    #elif [ "$__buildArch" == "armel" ]; then
+    #    buildRootfsCmd="/opt/corefx/cross/build-rootfs.sh armel tizen"
+    #else
+    #    echo "ERROR: unknown buildArch $__buildArch"
+    #    exit 1
+    #fi
+    #$dockerCmd $buildRootfsCmd
+
+    # Build rootfs in host
+    if [ "$__buildArch" == "arm" ]; then
+        sudo apt-get update
+        sudo apt-get -y install  qemu qemu-user-static binfmt-support debootstrap
+        buildRootfsCmd="sudo ./cross/build-rootfs.sh arm xenial --skipunmount"
+    elif [ "$__buildArch" == "armel" ]; then
+        sudo apt-get update
+        sudo apt-get -y install libxml2-utils rpm2cpio
+        buildRootfsCmd="sudo ./cross/build-rootfs.sh armel tizen --skipunmount"
+    else
+        echo "ERROR: unknown buildArch $__buildArch"
+        exit 1
+    fi
+    $buildRootfsCmd
+    sudo chown -R $(id -u -n) cross/rootfs
+
+    # Cross building corefx with rootfs in Docker
+    buildNativeCmd="/opt/corefx/build-native.sh -buildArch=$__buildArch -$__buildConfig -- cross $__verboseFlag"
+    if [ "$__buildArch" == "arm" ]; then
+        dockerCmd="docker run -i --rm -v $currentWorkingDirectory:/opt/corefx -w /opt/corefx chcosta/dotnetcore:ubuntu1404_cross_prereqs_v1"
+        buildManagedCmd="/opt/corefx/build-managed.sh -$__buildConfig -buildArch=$__buildArch -RuntimeOS=ubuntu.16.04"
+    else
+        dockerCmd="docker run -i --rm -v $currentWorkingDirectory:/opt/corefx -w /opt/corefx t2wish/dotnetcore:ubuntu1404_cross_prereqs_v2"
+        buildManagedCmd="/opt/corefx/build-managed.sh -$__buildConfig -BuildPackages=false"
```

Comment:
Since there is a workaround code for `arm` but not for `armel` yet at https://github.com/dotnet/corefx/blob/master/external/runtime/runtime.depproj#L8-L9, we can't use `-buildArch` option for `armel` yet. This will be addressed by other issue by @jyoungyun.

We can remove workaround code and use `-buildArch` option for `armel`, if we generate `armel` nupkgs from CoreCLR later.

Until then  we will use this command to build managed for CoreFX as a workaround.